ansible/playbooks/raf/cdap/deploy_cdap.yml

```diff
@@ -43,6 +43,21 @@
   roles:
     - {role: raf/cdap/deploy}
 
+- name: Deploy CDAP-UI Configs
+  hosts:
+    - all:!loadbalancer-nodes
+  any_errors_fatal: true
+  vars_files:
+    - "../../../playbooks/hortonworks/playbooks/group_vars/all"
+    - "../../../playbooks/hortonworks/playbooks/group_vars/ambari-server"
+  vars:
+    - cdap_cluster_name: "{{ cluster_name  }}"
```

Comment:
these are not required as any host group can be made available by ansible hosts file. All host groups should go in that file.